raiden/tests/utils/factories.py

```diff
@@ -1024,6 +1024,7 @@ def make_transfers_pair(
 
         pair = MediationPairState(received_transfer, lockedtransfer_event.recipient, sent_transfer)
         transfers_pairs.append(pair)
+        payer_index += 1
```

Comment:
Is this necessary? you are inside a range in a for, i mean is this ok? I think this could jump and skip indexes, the for starts like this ```for payer_index in range(number_of_channels - 1):``` lets suppose that the first loop payer_index = 0 then with this sentence you will put that payer_index = 1 before ending the loop 0, then the loop 1 will increase that value and instead of 1 you will have 2, is this possible or i'm just wrong?

Reply:
This is confusing for me. If the idea is to access to two contiguous positions, why not just use only one index? 

`channel_state=channels.get_sub_channel(payee_index +1)`

Reply:
As you guys say, the idea of this make_transfers_pair is to get access to two contiguos sub_channels.
I changed this cause the other way that i was implementing was more clearly than this one.